components/script/dom/htmliframeelement.rs

```diff
@@ -491,6 +490,18 @@ impl HTMLIFrameElementMethods for HTMLIFrameElement {
         element.set_bool_attribute(&local_name!("mozbrowser"), value);
     }
 
+    // https://html.spec.whatwg.org/multipage/#attr-iframe-allowfullscreen
+    fn AllowFullscreen(&self) -> bool {
```

Comment:
These should use `make_bool_getter!(AllowFullscreen, "allowfullscreen")` instead, due to https://html.spec.whatwg.org/multipage/embedded-content.html#dom-iframe-allowfullscreen .